.github/workflows/arduino.yml

```diff
@@ -0,0 +1,25 @@
+name: Arduino CI
+
+on: [push]
+jobs:
+  build-sketch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v2
+      - name: Install Arduino CLI
+        uses: arduino/setup-arduino-cli@v1.0.0
+      - name: Get Python3
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: Fetch sketch dependencies
+        run: |
+           pip3 install pyserial
+           arduino-cli core update-index --additional-urls "https://dl.espressif.com/dl/package_esp32_index.json"
+           arduino-cli core install esp32:esp32 --additional-urls "https://dl.espressif.com/dl/package_esp32_index.json"
+           arduino-cli lib install LIBRARY[@6.0.0](S) [flags]
+           arduino-cli lib install "Smartcar shield"
```

Comment:
If `arduino-cli lib install AudioZero@1.0.0` installs version `1.0.0` of Library `"AudioZero"` how should it be adjusted for version `6.0.0` of `"Smartcar shield"`?

Reply:
> You mean that I should create new arduinoSrc/WebServer (but called something else) that we have on our code and change it to new version

No. Please focus on line 22 of what you already have (and remove line 21 since its wrong). Considering the example I already mentioned, how would you download version `6.0.0` of the library `"Smartcar shield"`?